### PR TITLE
docs: fix broken html for deployment.md

### PR DESCRIPTION
**Pull request by JohannesHoppe**
minor fix in the markdown. without the closing quote the text and link are not displayed correctly

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

Content on angular.io is not shown. First part of the text is invisible and the link is not clickable.

## What is the new behavior?

Content on angular.io is shown.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/deployment.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ Make a note of the user name and project name in GitHub.
 
 You can see your deployed page at `https://<user_name>.github.io/<project_name>/`.
 
-<div class="alert is-helpful>
+<div class="alert is-helpful1">
```
Comment (brandonroberts) [inline aio/content/guide/deployment.md]: This should be `is-helpful`

Comment (brandonroberts) [inline aio/content/guide/deployment.md]: `is-helpful1` -> `is-helpful`

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: aio/content/guide/deployment.md (hunk 1 above)
Comment (JohannesHoppe) [reply aio/content/guide/deployment.md]: sorry, fat fingers! 😄

**Review by brandonroberts — APPROVED** [review]

**Review by IgorMinar — APPROVED** [review]
Thanks

**Comment by IgorMinar** [comment]
Caretaker note: legacy pullapprove flake - please ignore.

**Pull request closed without merging**